nodes/stampzilla-example/main.go

```diff
@@ -64,6 +43,34 @@ func main() {
 			"on": false,
 		},
 	}
+
+	node.OnRequestStateChange(func(state models.DeviceState, device *models.Device) error {
+		logrus.Info("OnRequestStateChange:", state, device.ID)
+
+		if device.ID == "1" {
+			dev3.State["on"] = state["on"]
+			node.AddOrUpdate(dev3)
```

Comment:
why `node.AddOrUpdate(dev3)` ? pretty sure its automatic if you dont return error...